CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Fixed
 - Fixed a bug where the asset ID was not being set correctly for Gwei and Wei
+- Added `rlp_encoded_tx` field to `EthereumTransaction` interface
```

Comment:
**nit**: This isn't really a fix, but more so an addition.

We should also use the imperative tense (e.g. `Fixed a bug...` should be `Fix bug...`)

```suggestion

### Added
- Adds a `rlp_encoded_tx` field to `EthereumTransaction` content
```

Reply:
Ahh missed the section. Will fix. Also I think "Add a" might be more appropriate than a "Adds a"  ?